feedback_report_generator/README.md

```diff
@@ -0,0 +1,25 @@
+create feedback reports
+=================
+Program to create feedback reports.
+
+### Configuration Settings
+
+Before running the script, set the following environment variables.
+
+| Variable                            | Default                | Description                                              |
+| ----------------------------------- | ---------------------- | -------------------------------------------------------- |
+| AzureCosmosDbDatabaseId | retrieve from portal | The id of a CosmosDB instance that holds the latest ukrlp data in a container called urkrlp.|
+| AzureCosmosDbUkRlpCollectionId | feedback | the container name holding the ukrlp data |

```

Comment:
Surely this is the feedback collection id?

Reply:
👍